www_files/app.py

```diff
@@ -34,6 +34,9 @@ def application(environ, start_response):
     cp.config.update(mmeds.config.CONFIG)
     cp.server.unsubscribe()
 
+    print("sssaaaaaa")
+    print("hello there")
+    print("yooooooooooo")
```

Comment:
I think either more helpful messages or no messages ...

the "hello there" can stay though. That's a nice greeting

Reply:
Oh yeah, The problem is that this is the only file apache on minerva directly interacts with. So sometimes it won't load changes to mmeds unless something in this file is modified. So usually I just throw some print statements in